wordpress-mobile/gutenberg-mobile
# Fix for video block crash on drawing on Android

This PR is a quick fix for wordpress-mobile/WordPress-Android#9943 
Makes sure to not set background color on Android.

To test:
See details WordPress/gutenberg#15857

Update release notes:

[ x ] If there are user facing changes, I have added an item to RELEASE-NOTES.txt.
